packages/react/src/components/dropdown/select/Select.tsx

```diff
@@ -526,6 +544,16 @@ export const Select = <OptionType,>(props: SelectProps<OptionType>) => {
           {getButtonLabel()}
           <IconAngleDown className={styles.angleIcon} aria-hidden />
         </button>
+        {showClearButtonForSingleSelect && (
+          <button
```

Comment:
Could we use the SelectedItems' clear button instead of duplicate button?

Reply:
It seemed to differ a bit. Maybe we could use the same css class atleast?

Reply:
It is pretty close at least. It would be easier to maintain if we did not have two different buttons. But if it is difficult, we could have one button for both cases.

Reply:
- [ ] Refactor to use only clear button from SelectedItems